src/interface/ICustomERC721.sol

```diff
@@ -106,6 +107,10 @@ interface ICustomERC721 {
   /// @notice Getter for the mint interval
   function MINT_INTERVAL() external view returns (uint256);
 
+  // TODO: Do we need this?
+  // /// @notice Getter for the minter role
+  // function minter() external view returns (address);
+
   /// @notice Admin function to update the sales configuration settings
```

Comment:
I'm not sure if we need this in the interface and what visibility we should give it. Solidity will automatically create a getter for public storage vars, but maybe to be consistent we add a function. What do you think @Yashiru ? I mostly just want to stay consistent

Reply:
I always include explicit function definitions in the interface for all callable methods. This ensures that other devs can interact with our contract through `ICustomERC721(address).anyViewFunction()` without needing to import the full `CustomERC721` contract in their projects.

Reply:
okay awesome. very good point